### DISCUSSION TITLE
[431960]wallpaper engine {小红车}

### DISCUSSION THREAD

**Discussion by beefdinner12** [pjy612/SteamManifestCache#583] [Share]
### Steam 游戏ID

`appid:431960`

### 密钥列表

```
431961;62ac71786b9fbeecc473fccb8a14da04ea676371addd8820638ee6cddac8c344
```

### 清单附件

[431961_3682614711587149074.zip](https://github.com/pjy612/SteamManifestCache/files/14960738/431961_3682614711587149074.zip)

### 备注

小红车应该不用我多说了吧，注意身体就行  2.4最新版本
邮箱shencixu02@163.com

**Comment by pjy612** [comment]
* 密钥格式 请参考 #26 
* 理论上已重复分享

**Reply by pjy612** [reply]
而且 默认不能连工坊 作用不大 推荐 入正
